.github/workflows/deploy.yml

```diff
@@ -0,0 +1,90 @@
+name: deploy
+
+on:
+  push:
+    branches: [master]
+    tags: ['v*']
+
+jobs:
+  dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: /home/runner/work/hathor-explorer/hathor-explorer/node_modules
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - name: Install Dependencies
+        run: |
+          npm install
+          tar -cvf node_modules.tar ./node_modules
+      - name: Upload node modules
+        uses: actions/upload-artifact@v2
+        with:
+          name: node_modules
+          path: node_modules.tar
+          if-no-files-found: error
+          retention-days: 1
+  deploy-testnet-explorer:
+    if: github.ref == 'refs/heads/master'
+    needs: dependencies
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download node modules
+        uses: actions/download-artifact@v2
+        with:
+          name: node_modules
+      - name: Build
+        run: |
+          tar -xf node_modules.tar
+          npm run build
+        env:
+          REACT_APP_BASE_URL: https://node1.foxtrot.testnet.hathor.network/v1a/
+          REACT_APP_WS_URL: wss://node1.foxtrot.testnet.hathor.network/v1a/ws/
+      - name: Deploy Testnet Explorer
+        run: |
+          make check_version
+          aws s3 sync --delete ./build/ s3://hathor-testnet-foxtrot-public-explorer
```

Comment:
Maybe we could create a make command for this script if I want to manually trigger an update on the testnet.  We can also create the make command for the mainnet.

Reply:
That's ok for me, will do it.